tcms/xmlrpc/serializer.py

```diff
@@ -214,10 +265,10 @@ def _get_m2m_fields(self):
         :rtype: list
         """
         if hasattr(self.__class__, 'm2m_fields'):
-            return self.__class__.m2m_fields
-        else:
-            return tuple(field.name for field in
-                         self.model_class._meta.many_to_many)
+            return self.__class__.m2m_fields  # pylint: disable=no-member
```

Comment:
Do not disable this. File an issue with pylint-django and this code snippet as example so I can fix it